pkg/support/support.go

```diff
@@ -108,7 +111,13 @@ func (bb *bundleBuilder) GenerateSupportBundle(additionalFiles []string, exclude
 			bb.logger.Infof("Excluding %s...", file)
 			continue
 		}
-		err := bb.copyInFile(writer, file, path.Join(zipPrefix, logPrefix))
+		bb.logger.Debugf("Copying %s...", file)
+		var err error
+		if isFileFromVM(file) {
+			err = bb.streamFileFromVM(writer, file, path.Join(zipPrefix, logPrefix))
+		} else {
+			err = bb.copyInFile(writer, file, path.Join(zipPrefix, logPrefix))
+		}
```

Comment:
nitpicky but could this be wrapped in a small util function, since this logic is repeated 2 other times?